mifospay/src/main/java/org/mifospay/savedcards/presenter/CardsScreenViewModel.kt

```diff
@@ -104,25 +95,17 @@ class CardsScreenViewModel @Inject constructor(
         mUseCaseHandler.execute(addCardUseCase, requestValues,
             object : UseCase.UseCaseCallback<AddCard.ResponseValue> {
                 override fun onSuccess(response: AddCard.ResponseValue) {
-                    onValidationSuccess(Constants.CARD_ADDED_SUCCESSFULLY)
+                    _cardState.value = CardsUiState.Success(Constants.CARD_ADDED_SUCCESSFULLY)
```

Comment:
can we use enum type constant instead of Constant in this case, for better code practise, what you say? 
And show toast message in screen using stringResource according to enum only. 
We don't need to use constant then.